AlertaDengue/locale/es/LC_MESSAGES/django.po

```diff
@@ -271,130 +270,168 @@ msgstr ""
 msgid "Nível Verde"
 msgstr ""
 
-#: dados/templates/alert_base.html:150
+#: dados/templates/alert_base.html:161
 msgid ""
 "Sem Risco. Quando o mapa está verde, não existe transmissão e nem condições "
 "ideais para a reprodução do Mosquito."
 msgstr ""
 
-#: dados/templates/alert_base.html:157
+#: dados/templates/alert_base.html:173
 msgid "Nível Amarelo"
 msgstr ""
 
-#: dados/templates/alert_base.html:161
+#: dados/templates/alert_base.html:183
 #, python-format
 msgid ""
 "Estação da %(disease_label)s. No nível amarelo, temos condições climáticas "
 "propícias à reprodução do mosquito. Mas ainda sem transmissão significativa "
 "da Doença."
 msgstr ""
 
-#: dados/templates/alert_base.html:168
+#: dados/templates/alert_base.html:195
 msgid "Nível Laranja"
 msgstr ""
 
-#: dados/templates/alert_base.html:172
+#: dados/templates/alert_base.html:204
 #, python-format
 msgid ""
 "Transmissão de %(disease_label)s. No nível laranja, já observamos uma "
 "incidência pronunciada da %(disease_label)s, mas ainda não podemos afirmar "
 "com confiança que teremos epidemia."
 msgstr ""
 
-#: dados/templates/alert_base.html:179
+#: dados/templates/alert_base.html:216
 msgid "Nível Vermelho"
 msgstr ""
 
-#: dados/templates/alert_base.html:183
+#: dados/templates/alert_base.html:225
 #, python-format
 msgid ""
 "Epidemia. Neste nível estamos em plena epidemia. Todo o cuidado é pouco, e "
 "precisamos ficar atentos aos sintomas da %(disease_label)s e procurar o "
 "médico imediatamente em caso suspeita de %(disease_label)s."
 msgstr ""
 
-#: dados/templates/alert_base.html:192 dados/templates/alert_state/map.html:212
+#: dados/templates/alert_base.html:234 dados/templates/alert_state/map.html:212
 msgid "Casos estimados"
 msgstr ""
 
-#: dados/templates/alert_base.html:194
+#: dados/templates/alert_base.html:236
 #, python-format
 msgid ""
 " São estimados entre <b>%(min_est)s</b> e <b>%(max_est)s novos casos</b> na "
 "Semana Epidemiológica %(SE)s: %(data1)s  a %(data2)s. Com %(total_observed)s "
 "casos notificados oficialmente até o momento"
 msgstr ""
 
-#: dados/templates/alert_base.html:195
+#: dados/templates/alert_base.html:237
 #, python-format
 msgid " Últimas %(look_back)s semanas: "
 msgstr ""
 
-#: dados/templates/alert_base.html:205
+#: dados/templates/alert_base.html:247
 msgid "Suspeita de um criadouro?"
 msgstr ""
 
-#: dados/templates/alert_base.html:206
-msgid ""
-"Em caso de suspeita de criadouro de mosquitos, ligue para a <a href=\"http://"
-"www.1746.rio.gov.br/\" style=\"color: #FFFFFF\">Central de atendimento ao "
-"cidadão 1746</a>."
+#: dados/templates/alert_base.html:249
+msgid "Em caso de suspeita de criadouro de mosquitos, "
 msgstr ""
 
-#: dados/templates/alert_base.html:216
+#: dados/templates/alert_base.html:253
+msgid "contacte a "
+msgstr ""
+
+#: dados/templates/alert_base.html:255
+msgid "central de apoio ao cidadão: 1746."
+msgstr "póngase en contacto con la central de apoyo al ciudadano: 1746"
```

Comment:
fix the translation, please.

Reply:
ok: central de apoyo al ciudadano: 1746.